Replace soelim command from heavy port groff with lightweight mandoc.

--- build/top.mk.orig	2024-05-21 11:19:11
+++ build/top.mk	2024-11-21 16:43:14
@@ -153,7 +153,7 @@
 MANCOMPRESS=$(CAT)
 MANCOMPRESSSUFFIX=
 
-SOELIM=soelim
+SOELIM=msoelim
 
 INCLUDEDIR= $(top_srcdir)/include
 LDAP_INCPATH= -I$(LDAP_INCDIR) -I$(INCLUDEDIR)
